README.md

```diff
@@ -29,7 +29,7 @@ provider "helm" {
 ```
 module "kube_prometheus_stack" {
   source = "sparkfabrik/terraform-sparkfabrik-prometheus-stack"
-  prometheus_stack_chart_version = "32.0.0"
+  prometheus_stack_chart_version = "31.0.0"
```

Comment:
Why are we downgrading to 31 ?

Reply:
Because the version 32 has not yet been released